packages/widget-markdown/package.json

```diff
@@ -6,10 +6,12 @@
   "main": "./build/index.js",
   "dependencies": {
     "@fortawesome/react-fontawesome": "^0.1.17",
+    "@types/node-fetch": "2",
     "@types/react-splitter-layout": "^3.0.2",
     "@types/react-virtualized": "^9.21.20",
     "@vscode-marquee/utils": "^0.1.0",
     "@vscode-marquee/widget": "^0.1.0",
+    "node-fetch": "2",
```

Comment:
Using node-fetch 2 because of [this: #nodefetch-](https://github.com/node-fetch/node-fetch/issues/1266). We could also use an alternative like [superagent](https://github.com/visionmedia/superagent).

Reply:
lets define the version as `^2.6.7`